README.md

```diff
@@ -23,6 +24,229 @@ Terra Toolkit is a utility module used to facilitate independent development of
+
+const config = {
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  axe: {
+    // Don't inject axe script, its included in test files
+    inject: false,
+  },
+  ...wdioConf.config,
+};
+
+exports.config = config;
+```
+
+
+#### Usage
+
+`browser.axe([{options}]);`
+
+The following options are available:
+
+* **viewports**:
+  An array of viewports `{ width, height }` to run the accessibility test in. If none provided uses the current viewport.
+* **rules**:
+  The axe rules configuration to test. See [axe documentation](https://www.axe-core.org/docs/)
+* **runOnly**:
+  The axe tags to filter the validations to run on the accessibility to test. See [axe documentation](https://www.axe-core.org/docs/)
+* **context**:
+  A css selector to scope the accessibility test to. See [axe documentation](https://www.axe-core.org/docs/)
+
+#### Examples
+```js
+it('ignores inaccessibility based on rules', () => {
+  browser.url('/inaccessible-contrast.html');
+  const rules = {
+    'color-contrast': { enabled: false },
+  };
+  expect(browser.axe({ viewports, rules })).to.be.accessible();
+});
+
+it('runs only specified tags', () => {
+  browser.url('/inaccessible-text.html');
+  const runOnly = {
+    type: 'tag',
+    values: ['color-contrast'],
+  };
+  expect(browser.axe({ viewports, runOnly })).to.be.accessible();
+});
+
+it('runs only specified context', () => {
+  browser.url('/inaccessible-contrast.html');
+  let context = 'h1';
+  expect(browser.axe({ viewports, context })).to.not.be.accessible();
+
+  context = 'h2';
+  expect(browser.axe({ viewports, context })).to.be.accessible();
+});
+```
+
+### Visual Regression Tests
+Terra toolkit provides configuration for a visual regression utility. See [documentation](http://webdriver.io/guide/services/visual-regression.html) for more information. Terra Toolkit provides useful wrappers to make setup easier.
+
+Visual regression tests can be written using the commands provided by the [visual regression service](http://webdriver.io/guide/services/visual-regression.html). Example:
+
+```js
+/* global browser, describe, it, expect, viewport */
+describe('button test', () => {
+  // Only test tiny/huge viewports
+  const viewports = viewport('tiny', 'huge');
+  it('checks visual comparison', () => {
+    browser.url('/buttons.html');
+
+    // Verify the button element matches
+    let screenshots = browser.checkElement('button', { viewports });
+    expect(screenshots).to.matchReference();
+
+    // Verify the whole viewport matches
+    let screenshots = browser.checkViewprot({ viewports });
+    expect(screenshots).to.matchReference();
+  });
+});
+```
+
+### Test Helpers
+
+The terra toolkit provides custom assertions and globals to make testing easier.
+
+#### Viewports
+
+The `viewports()` function provides a convenience method for fetching an array of viewports to use to resize the browser or even pass directly to the accessibility and visual regression commands.
```

Comment:
In the code, this is named viewport(). IMO viewports() would is what we want

Reply:
not against that, but then the challenge with that is then you can't do this:
```js
// ERROR: viewports is already defined
const viewports = viewports('tiny');
```
Having the variable named viewports is nice cause then you can use object shorthand syntax rather than saying `{ viewports: viewports }` one can simply do `{ viewports }`

Reply:
^ what you say makes sense. That sorta makes me want to change the name of `viewport()` to something else then since it doesn't read like it would return multiple viewport sizes. Possible ideas could be 'screenSizes()', 'viewportSizes()', terraViewports(), screenDimensions().

Either way, the documentation is incorrect. 
> The `viewports()` function

should be updated to The _viewport_() function